diff -Naur media_build-2017-06-20-rpi.orig/v4l/compat.h media_build-2017-06-20-rpi/v4l/compat.h
--- media_build-2017-06-20-rpi.orig/v4l/compat.h	2017-07-17 18:39:36.067668799 +0200
+++ media_build-2017-06-20-rpi/v4l/compat.h	2017-07-17 18:40:46.375154630 +0200
@@ -1588,7 +1588,7 @@
 #endif
 
 #ifdef NEED_WRITEL_RELAXED
-#define writel_relaxed writel
+// #define writel_relaxed writel
 #endif
 
 #ifdef NEED_GET_USER_PAGES_UNLOCKED
diff -Naur media_build-2017-06-20-rpi.orig/v4l/versions.txt media_build-2017-06-20-rpi/v4l/versions.txt
--- media_build-2017-06-20-rpi.orig/v4l/versions.txt	2017-07-17 18:39:36.067668799 +0200
+++ media_build-2017-06-20-rpi/v4l/versions.txt	2017-07-17 18:42:03.334594331 +0200
@@ -1,5 +1,6 @@
 # Use this for stuff for drivers that don't compile
 [9.255.255]
+VIDEO_S5C73M3
 INTEL_ATOMISP
 
 [4.12.0]
